webpack/webpack.config.dev-server.js

```diff
@@ -1,38 +1,43 @@
 var path = require('path');
 var webpack = require('webpack');
-var commonLoaders = require('./common.config').commonLoaders;
-var assetsPath = require('./common.config').assetsPath;
-var publicPath = require('./common.config').publicPath;
+
+var commonConfig = require('./common.config');
+
+var commonLoaders = commonConfig.commonLoaders;
+var publicPath = commonConfig.output.publicPath;
+var externals = commonConfig.externals;
 
 module.exports = {
```

Comment:
I'm not seeing this either. I do get all the stylesheets and some 'undefineds' output from `npm run dev`, so not sure if something funny is going on there

```
Hash: f81dce7a923cc216b717
Version: webpack 1.14.0
Time: 6379ms
        Asset    Size  Chunks             Chunk Names
server.dev.js  873 kB       0  [emitted]  server
    + 309 hidden modules
--------------------------
===> 😊  Starting Server . . .
===>  Environment: development
===>  Listening on port: 3000
===>  Using DB TYPE: MONGO
--------------------------
===>  Succeeded in connecting to mongodb://localhost/ReactWebpackNode

webpack building...

app/css/components/about.css
undefined [undefined]


app/css/components/navigation.css
undefined [undefined]
undefined [undefined]
```